.travis.yml

```diff
@@ -15,7 +16,8 @@ before_script:
   - export PATH=$HOME/.local/bin:$PATH
 script:
   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-  - julia --check-bounds=yes -e 'Pkg.clone(pwd()); Pkg.build("BioServices"); Pkg.test("BioServices"; coverage=true)'
+  # - julia -e 'Pkg.clone(pwd(), "BioServices"); Pkg.build("BioSercices"); ENV["TRAVIS"] = "yes"; Pkg.test("BioServices"; coverage=true)'
+  - julia --check-bounds=yes -e 'import Pkg; Pkg.build(); Pkg.test("BioServices"; coverage=true)'
 after_success:
-  - julia -e 'cd(Pkg.dir("BioServices")); Pkg.add("Coverage"); using Coverage; Codecov.submit(process_folder())'
-  - julia -e 'cd(Pkg.dir("BioServices")); Pkg.add("Documenter"); include(joinpath("docs", "make.jl"))'
+  - julia -e 'import Pkg; Pkg.instantiate(); Pkg.add("Documenter"); Pkg.add("Literate"); include(joinpath("docs", "make.jl"))'
```

Comment:
1. I don't think we are using Literate in this package.
2. I think the `script:` section can be left empty and it will use the default tests. Remove un-used code
3. Is `Pkg.instantiate()` necessary?